srcpkgs/setzer/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'setzer'
+pkgname=setzer
+version=0.3.2
+revision=1
+wrksrc=Setzer-${version}
+build_style=meson
+hostmakedepends="gettext"
+makedepends="python3-gobject-devel"
+depends="poppler-glib virtual?tex gspell python3-xdg python3-gobject gtksourceview4"
+short_desc="Simple yet full-featured LaTeX editor written in Python with Gtk"
+maintainer="Bleznudd <bleznudd@pm.me>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/cvfosammmm/setzer"
+distfiles="https://github.com/cvfosammmm/setzer/archive/v${version}.tar.gz"
```

Comment:
Since you're going to have to push to retrigger CI, would you mind changing this to `distfiles="${homepage}/...`?

Reply:
Sure